crates/core/component/shielded-pool/src/ics20_withdrawal.rs

```diff
@@ -1,4 +1,8 @@
-use ibc_types::core::{channel::ChannelId, channel::PortId, client::Height as IbcHeight};
+use ibc_types::core::{channel::ChannelId, client::Height as IbcHeight};
+
+#[cfg(feature = "component")]
+use ibc_types::core::channel::PortId;
```

Comment:
This change makes me suspect I'm optimizing for the disabling warnings, rather than meaningfully structuring the imports.